template.yml

```diff
@@ -206,9 +207,18 @@ Resources:
             Value: "true"
         PointInTimeRecoverySpecification:
           PointInTimeRecoveryEnabled: true
+  
+  RunPluginDdbGlobalTables:
+    Type: Custom::GlobalTable
+    Condition: IsSecondaryRegion
+    Properties:
+      ServiceToken:
+        Ref: DdbGlobalTableArn
```

Comment:
you could use `ServiceToken: !Ref DdbGlobalTableArn` here for consistency and brevity.

Reply:
updated